include/ignition/rendering/Utils.hh

```diff
@@ -30,6 +35,50 @@ namespace ignition
     // Inline bracket to help doxygen filtering.
     inline namespace IGNITION_RENDERING_VERSION_NAMESPACE {
     //
+    /// \brief Retrieve the first point on a surface in the 3D scene hit by a
+    /// ray cast from the given 2D screen coordinates.
+    /// \param[in] _screenPos 2D coordinates on the screen, in pixels.
+    /// \param[in] _camera User camera
+    /// \param[in] _rayQuery Ray query for mouse clicks
+    /// \param[in] _maxDistance maximum distance to check the collision
+    /// \return 3D coordinates of a point in the 3D scene.
+    IGNITION_RENDERING_VISIBLE
+    math::Vector3d ScreenToScene(
```

Comment:
global functions start with lower case. Can we change this to `screenToScene`? Same goes for the other functions added in this class.

Reply:
ahh I'm a few mins late

Reply:
https://github.com/ignitionrobotics/ign-rendering/pull/368